pages/user/edit.tsx

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-console */
+/* eslint-disable spaced-comment */
+import React from 'react'
```

Comment:
For some reason the placeholders in the input components of this file are not properly centerd